bin/bashrc-functions.sh

```diff
@@ -4,16 +4,16 @@
 
 register_die_hook() {
 	local x
-	for x in $* ; do
-		has $x $EBUILD_DEATH_HOOKS || \
+	for x in "$@" ; do
+		has "$x" "$EBUILD_DEATH_HOOKS" || \
```

Comment:
That won't work because EBUILD_DEATH_HOOKS is a space separated list, and has() expects a separate argument for each element.